mnemosyne/core/binary_vectors.py

```
@@ -20,9 +20,8 @@
 """
 
 import numpy as np
-import json
 import sqlite3
-from typing import List, Dict, Tuple, Optional
+from typing import List, Dict
```

Ruff (0.15.21)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)